messytables/util.py

```diff
@@ -59,7 +59,7 @@ def __reduce__(self):
         items = MutableMapping.items
 
         def __repr__(self):
-            pairs = ', '.join(map('%r: %r'.__mod__, self.items()))
+            pairs = ', '.join(map('%r: %r'.__mod__, list(self.items())))
```

Comment:
On the one hand, this `list()` is unnecessary (but I did have to check that)...

On the other hand, should probably use `.format()`:

```
', '.join("{!r}: {!r}".format(*item) for item in self.items())
```